<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TTCutListWidget</class>
 <widget class="QWidget" name="TTCutListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>255</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>16</width>
    <height>104</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>1</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="horizontalSpacing">
    <number>8</number>
   </property>
   <property name="verticalSpacing">
    <number>4</number>
   </property>
   <item row="0" column="0" colspan="6">
    <widget class="QTreeWidget" name="videoCutList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>120</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <attribute name="headerStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Videofile</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cut-In</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cut-Out</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Length</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>New Cut-Out</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Audiooffset</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Duration</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Size</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Total Duration</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Total Size</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pbEntryUp">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="audiofilelistwidget.qrc">
       <normaloff>:/pixmaps/pixmaps/uparrow_18.xpm</normaloff>:/pixmaps/pixmaps/uparrow_18.xpm</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pbEntryDown">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="audiofilelistwidget.qrc">
       <normaloff>:/pixmaps/pixmaps/downarrow_18.xpm</normaloff>:/pixmaps/pixmaps/downarrow_18.xpm</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pbEntryDelete">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="audiofilelistwidget.qrc">
       <normaloff>:/pixmaps/pixmaps/cancel_18.xpm</normaloff>:/pixmaps/pixmaps/cancel_18.xpm</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>250</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="pbPreview">
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>Preview</string>
     </property>
     <property name="icon">
      <iconset resource="ttsettingsdialog.qrc">
       <normaloff>:/pixmaps/pixmaps/preview_18.xpm</normaloff>:/pixmaps/pixmaps/preview_18.xpm</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>14</width>
       <height>14</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="pbCutAudioVideo">
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>Cut</string>
     </property>
     <property name="icon">
      <iconset resource="cutlistwidget.qrc">
       <normaloff>:/pixmaps/pixmaps/cutav_18.xpm</normaloff>:/pixmaps/pixmaps/cutav_18.xpm</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>14</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ttsettingsdialog.qrc"/>
  <include location="audiofilelistwidget.qrc"/>
  <include location="cutlistwidget.qrc"/>
 </resources>
 <connections/>
</ui>
